test/resources/jenkins_2.141/credentials.xml

```diff
@@ -24,6 +24,13 @@
           </privateKeySource>
         </com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey>
       </java.util.concurrent.CopyOnWriteArrayList>
+      <list>
+        <com.example.ExampleCredential>
+          <id>exampleId</id>
+          <username>exampleUsername</username>
+          <password>examplePassword</password>
+        </com.example.ExampleCredential>
+      </list>
```

Comment:
Hey @rbjorklin  , thanks for the PR again :)
I've deleted the PR notification email by accident two weeks ago without noticing it, sorry about that...
I've made myself better email filters now to not miss PRs and issues.

Looks very good. I've let myself add an example and a test for to this PR.

Could you just verify that a `<list>` example is something like the above?  
Does the xpath look ok (is it in the right place) and the names are ok?  
Best would be if you could replace the above with a "realistic" example (copied from actual jenkins with modified secrets values) to be 100% sure it's gonna work.

If this example looks good I'll merge the PR.